### PR TITLE
[DLRS v0.6.0] dlrs/clearlinux: Pytorch and ML compiler update

### PR DESCRIPTION
**Pull request by DnPlas**
This commit updates the following:
* Pytorch
* ML Compiler

Signed-off-by: Daniela Plascencia <daniela.plascencia@linux.intel.com>

### CODE REVIEW

File: dlrs/clearlinux/pytorch/mkl/Dockerfile [hunk 1]
```diff
@@ -1,65 +1,74 @@
...
-    && rm -rf /var/lib/swupd/*
-# copy and execute installation scripts
-COPY ./scripts /scripts
-RUN cd /scripts \
-  && ./install_conda.sh \
-  && ./install_torch_deps.sh \ 
-  && ./install_pytorch.sh \
-  && ./install_torchvision.sh \
-  && ./install_utils.sh \
-  && cd / && rm -rf /scripts \
```
Comment (unrahul) [inline dlrs/clearlinux/pytorch/mkl/Dockerfile:20]: phew :)

File: dlrs/clearlinux/pytorch/mkl/licenses/clear_LICENSE [hunk 2]
```diff
@@ -43,6 +43,7 @@ BSD-3-Clause-LBNL
 BSD-4-Clause
 BSD-4-Clause-UC
 BSL-1.0
+Beerware
```
Comment (unrahul) [inline dlrs/clearlinux/pytorch/mkl/licenses/clear_LICENSE:46]: Thats first time i have heard about this License, from now on gonna use that instead for my projects :D

**Review by unrahul — APPROVED** [review]
LGTM, thank you @DnPlas

**Comment by unrahul** [comment]
@DnPlas are we missing the pytorch-oss dockerfiles?

File: dlrs/clearlinux/pytorch/mkl/licenses/clear_LICENSE (hunk 2 above)
Comment (DnPlas) [reply dlrs/clearlinux/pytorch/mkl/licenses/clear_LICENSE:46]: Hahaha, cool license, isn't it?

**Comment by DnPlas** [comment]
@unrahul not missing, nothing changed in the pytorch-oss Dockerfile,that's why it is not there

**Comment by unrahul** [comment]
> @unrahul not missing, nothing changed in the pytorch-oss Dockerfile,that's why it is not there

early in the day, I need more coffee :D

**Pull request merged**